modules/sildisco/src/Auth/Process/LogUser.php

```diff
@@ -65,9 +65,9 @@ public function __construct(array $config, mixed $reserved)
     /**
      * Log info for a user's login to Dynamodb
      *
-     * @param array &$state  The current state array
```

Comment:
```suggestion

```

Reply:
The "Log info for a user's login to Dynamodb" description is for this class, not the Auth/Process class. Combining this phrase with @inheritDoc provides both descriptions in the function documentation, as best I understand.

Reply:
Ah, ok, thanks!